lab-abigail/lib/lab-02.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+module.exports = exports = {};
+
+exports.Animal = function(name, breed, age, city) {
+  this.name = name;
+  this.breed = breed;
+  this.age = age;
+  this.city = city;
+};
+
+//Call
```

Comment:
Comments are great! Try to manage a different branch that contains your comments and submit a clean branch to me. I'll start docking points next week.